torch/_dynamo/variables/nn_module.py

```diff
@@ -488,6 +488,11 @@ def call_function(
                 tx.output.is_root_tracer()
                 and mod.__module__.startswith(("torch.nn.", "torch.ao."))
                 and mod.__module__ != "torch.nn.utils.parametrize"
+                # this basically means we are using the new strict export tracer which wraps the
+                # user callable, so we shouldn't directly proxy in the fx graph
+                and not isinstance(
+                    mod, torch.ao.quantization.pt2e.export_utils._WrapperModule
+                )
```

Comment:
Please please turn on `install_free_tensors` - everything inside `NNModuleVariable` is bitrotten. And when you turn on the flag (which is inevitable), you might see different set of issues, rendering all this work useless.